tests/run-with-compiler/reflect-sourceCode/Test_2.scala

```diff
@@ -0,0 +1,15 @@
+import api._
+
+object Test {
+  def f(implicit x: Int): Int = x * x
+  def main(args: Array[String]): Unit = {
+    implicit val x: Int = 10
+    assert(args(0).reflect == "args(0)")
+    assert(args( 0 ).reflect == "args( 0 )")
+    assert(args( 0 /* ignore */).reflect == "args( 0 /* ignore */)")
+    assert(f.reflect == "f")
+    assert((f).reflect == "f")
+    assert( { f }.reflect == "f")
+    assert( { f; f }.reflect == "f")
```

Comment:
@nicolasstucki I'm not sure if the behavior at line 13 is intended or not?

Reply:
It is not. I would check if the positions are correct.

Reply:
@liufengyun you should remove this test from here and create an issue. This way we can merge the other tests.